runtime/compiler/build/files/host/p.mk

```diff
@@ -45,11 +45,7 @@ ifeq ($(OS),aix)
     PPC_HW_PROFILER=compiler/p/runtime/PPCHWProfilerAIX.cpp
     PPC_LM_GUARDED_STORAGE=compiler/p/runtime/PPCLMGuardedStorageAIX.cpp
 
-    ifeq ($(I5_VERSION),I5_V6R1)
-        PPC_HW_PROFILER=
-        PPC_LM_GUARDED_STORAGE=
-    endif
-    ifeq ($(I5_VERSION),I5_V7R2)
+    ifneq ($(findstring $(I5_VERSION),I5_V6R1 I5_V7R2 I5_V7R3 I5_V7R4),)
```

Comment:
1) Is something missing after the last comma?
2) Will this line need to be modified with any new I5 version?

Reply:
It means NULL after the last comma.  The new changed line merged all determination branch.
If the value of I5_VERSION is in  I5_V6R1/I5_V7R2/I5_V7R3/I5_V7R4, set  PPC_HW_PROFILER and PPC_LM_GUARDED_STORAGE to NULL. @gt